Gemfile

```diff
@@ -16,6 +16,7 @@ gem "acts_as_follower", github: "thepracticaldev/acts_as_follower", branch: "mas
 gem "addressable", "~> 2.7" # A replacement for the URI implementation that is part of Ruby's standard library
 gem "administrate", "~> 0.12" # A Rails engine that helps you put together a super-flexible admin dashboard
 gem "ahoy_email", "~> 1.1" # Email analytics for Rails
+gem 'ahoy_matey' # Track visit/event
```

Comment:
this gem was added to track users and their visits to the site, we are not using it at the moment as we decided to go with the `sign_in_count` attribute to count the visits. My hope is that we can try to use this once we are done with the MVP

Reply:
thanks for looking into this! I agree that using the sign in count is a good way to get this metric for now, and we can revisit if time allows.